layouts/products/list.html

```diff
@@ -1,79 +1,51 @@
 
 <section class="work-in-progress--product-module border-{{.Params.phase}}"
          id="{{.Params.translationKey}}">
-  <!-- Product Title-->
-  <div class="row">
-    <div class="col-sm-10 col-sm-offset-1 col-xs-12">
-      <div class="title-container">
-        <div>
-          <h4 class='{{ .Params.phase }}{{ if eq (string (.Site.Language)) "fr" }}fr{{end}}'>
-            {{ .Params.title }}
-          </h4>
-        </div>
-      </div>
-    </div>
-  </div>
-  <!-- End Product Title-->
+  <h1 class='{{ .Params.phase }}{{ if eq (string (.Site.Language)) "fr" }}fr{{end}}'>
```

Comment:
love the removal of this nesting. There's an offset column originally, but then it was just countered via css? Strange. This is much nicer and cleaner, especially since we weren't even using the offset.

Reply:
Yea, I just added the padding back into the CSS for the section and it corrected the design. I'm not really sure why all the HTML was in there ....